pages/common/javac.md

```diff
@@ -5,3 +5,19 @@
 - Compile a .java file:
 
 `javac {{filename.java}}`
+
+- Compile two or more .java files:
+
+`javac {{filename1.java}} {{filename2.java}} {{filename3.java}}`
+
+- Compile all .java files in current directory:
+
+`javac {{*.java}}`
+
+- Compile a .java file and place class file in a specific directory:
+
+`javac -d {{path/to/some/directory}} {{filename.java}}`
+
+- Compile two or more .java files and place class files in a specific directory:
```

Comment:
This one is unnecessary IMO, but if you prefer to keep it, use "several" here too.